traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -369,6 +382,78 @@ func (inf *APIInfo) Close() {
 	}
 }
 
+// CreateInfluxClient onstructs and returns an InfluxDB HTTP client, if enabled and when possible.
+// The error this returns should not be exposed to the user; it's for logging purposes only.
+//
+// If Influx connections are not enabled, this will return `nil` - but also no error. It is expected
+// that the caller will handle this situation appropriately.
+func (inf *APIInfo) CreateInfluxClient() (c *influx.Client, e error) {
+	if !inf.Config.InfluxEnabled {
+		return
+	}
+
+	var fqdn string
+	var TCPPort uint
```

Comment:
Why? `go fmt` doesn't care

Reply:
just the convention I have seen for variable names in golang

Reply:
Yeah, local variables should start with lowercase, and be consistent-cased: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
That section indicates initialisms and abbreviations can be uppercased. I'll change `fqdn` to match.

If conventions like this are supposed to be followed, we should have a mechanism for checking them: some kind of linter.

Reply:
yes we should definitely be running a linter 💯

Reply:
>That section indicates initialisms and abbreviations can be uppercased. I'll change fqdn to match.

They can, but local variables should always start with lowercase, hence public and global symbols starting with initialisms should be e.g. `URLFoo`, and private and local should be `urlFoo`.

Local variables should always start with lowercase, because Go exports symbols starting with uppercase, and does not export lowercase. Of course, a local variable in a function can't be viewed from an outside scope. So it makes it very odd and confusing for a local var to be uppercase="exported." If nothing else, if a future reader sees that symbol uses further down in the func, and doesn't notice it's local, they're going to assume it's an exported global variable and be confused.

Reply:
I'll change it because I don't really care that much - though for my money "url" is nonsense that doesn't mean anything while "URL" stands for something - but I don't think it's fair to require a convention not agreed-upon by or enforced mechanically by the project. If local variables should start with a lowercase letters, there should be a linter configuration introduced to the repository and approved on the mailing list.